### PR TITLE
Visualization beginning

### PR DESCRIPTION
**Pull request by jspaaks**
This PR adds a more enticing beginning to the visualization blog.

### CODE REVIEW

File: show-me-the-visualization/README.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-
-Let's make changes to the C++ code to store the data from the iterations.
+![Under the hood](peugeot.jpg)
+_Visualization allows us to track what's going on under the hood of our program. Image courtesy of [Nenad Stojkovic](https://www.flickr.com/photos/nenadstojkovic/49044775348)._
 
 ## Iterations
 
-To store data of an iteration we will use a structure with the following variables:
+The Newton-Raphson algorithm approximates the value of the target equation's root in a series of iterations. We capture
+the data of each iteration using the following `struct` array:
```
Comment (jspaaks) [inline show-me-the-visualization/README.md]:
```suggestion
the data of each iteration using the following `struct`:
```

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Like the under the hood analogy.

As the root-plot.png has been replaced, please remove root-plot.png and root-plot.json from show-me-the-visualization/ dir.

**Comment by sverhoeven** [comment]
Is this your car?

**Comment by jspaaks** [comment]
> Is this your car?

I was tempted, but couldnt find a good pic quickly

**Review by sverhoeven — APPROVED** [review]

**Comment by jspaaks** [comment]
Thanks for reviewing!

**Pull request merged**